.gitlab-ci.yml

```diff
@@ -95,22 +96,150 @@ whois-integration-test:
     - mariadb:10.2
   tags:
     - whois
+  stage: integration-test
+  rules:
+    - if: '$CI_COMMIT_REF_NAME == "master"'
+      when: always
+    - when: manual
+      allow_failure: true
+  script:
+    - mvn -DskipTests install
+    - mvn $MAVEN_OPTS -Ddb.host=mariadb verify -Pintegration -pl whois-api -T 1C
+
+whois-client-integration-test:
+  <<: *it_env
+  services:
+    - mariadb:10.2
+  tags:
+    - whois
+  stage: integration-test
+  rules:
+    - if: '$CI_COMMIT_REF_NAME == "master"'
+      when: always
+    - when: manual
+      allow_failure: true
+  script:
+    - mvn -DskipTests install
+    - mvn $MAVEN_OPTS -Ddb.host=mariadb verify -PintegrationAll -pl whois-client -T 1
```

Comment:
Minor: Although it takes very less time to mvn install but is it possible to share this artifact path using a generic step